src/Microsoft.TestPlatform.ObjectModel/Microsoft.TestPlatform.ObjectModel.csproj

```diff
@@ -6,7 +6,8 @@
   <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
   <PropertyGroup>
     <AssemblyName>Microsoft.VisualStudio.TestPlatform.ObjectModel</AssemblyName>
-    <TargetFrameworks>net45;net451;netstandard2.0;netstandard1.3;netstandard1.0;uap10.0</TargetFrameworks>
+    <TargetFrameworks>net45;net451;netstandard2.0;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);uap10.0;netstandard1.0</TargetFrameworks>
```

Comment:
Any reason we are skipping NS1.0 build on Linux, is it broken like UAP?

Reply:
It seemed so.